votingpool/example_test.go

```diff
@@ -17,109 +17,262 @@
 package votingpool_test
 
 import (
+	"bytes"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 
 	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcd/txscript"
+	"github.com/btcsuite/btcutil"
+	"github.com/btcsuite/btcwallet/txstore"
 	"github.com/btcsuite/btcwallet/votingpool"
 	"github.com/btcsuite/btcwallet/waddrmgr"
 	"github.com/btcsuite/btcwallet/walletdb"
 	_ "github.com/btcsuite/btcwallet/walletdb/bdb"
 )
 
-func Example_basic() {
-	// This example demonstrates how to create a voting pool, create a
-	// series, get a deposit address from a series and lastly how to
-	// replace a series.
+var (
+	pubPassphrase  = []byte("pubPassphrase")
+	privPassphrase = []byte("privPassphrase")
+)
+
+func Example_depositAddress() {
+	// This example demonstrates how to create a voting pool with one
+	// series and get a deposit address for that series.
+
+	// Create the address manager and votingpool DB namespace.
+	mgr, vpNamespace, tearDownFunc, err := exampleCreateMgrAndDBNamespace()
```

Comment:
There probably needs to be an example which does what this function is doing along with a comment stating as much since the called function won't show up in the example documentation.